modules/images/webp-uploads/settings.php

```diff
@@ -33,10 +33,10 @@ function webp_uploads_add_media_settings_field() {
 	// Add settings field.
 	add_settings_field(
 		'perflab_generate_webp_and_jpeg',
-		__( 'Generate WebP and JPEG', 'performance-lab' ),
+		__( 'WebP and JPEG', 'performance-lab' ),
 		'webp_uploads_generate_webp_jpeg_setting_callback',
 		'media',
-		'uploads',
+		! is_multisite() ? 'uploads' : 'default',
```

Comment:
For better readability, let's reverse this.
```suggestion
		is_multisite() ? 'default' : 'uploads',
```